Source/Csla.Shared/Configuration/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿#if NETSTANDARD2_0
+//-----------------------------------------------------------------------
+// <copyright file="ConfigurationExtensions.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: http://www.lhotka.net/cslanet/
+// </copyright>
+// <summary>Implement extension methods for .NET Core configuration</summary>
+//-----------------------------------------------------------------------
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Csla.Configuration
+{
+  /// <summary>
+  /// Implement extension methods for .NET Core configuration
+  /// </summary>
+  public static class ConfigurationExtensions
+  {
+    /// <summary>
+    /// Add CSLA .NET services for use by the application
+    /// and enable CSLA .NET configuration.
+    /// </summary>
+    /// <param name="services">IServiceCollection object</param>
+    public static ICslaBuilder AddCsla(this IServiceCollection services)
+    {
+      services.AddSingleton<IDataPortalService, DataPortalService>();
+      return null;
```

Comment:
@rockfordlhotka 
What is an idea behind returning `null`. Maybe some default implementation of ICslaBuilder?

Reply:
That might be better. Right now I don't have anything to actually do in an ICslaBuilder - but that's the convention of what these methods should return. You are probably right though, it is probably better to return an empty class that implements the empty interface rather than null.